readme/Readme.scala

```diff
@@ -68,6 +68,7 @@ object Readme {
     val code1 = s"import scala.meta._$EOL${unindent(code0).trim}"
     val result0 = executeInRepl(code1)
     val result1 = result0.split(EOL).drop(3).mkString(EOL)
+    print(".")
```

Comment:
This isn't necessary, but it provides a progress bar for building the readme. ILoop is super slow! In scalameta/tutorial we have caching infrastructure which makes it incremental and super fast for subsequent runs. We should really move the readme project from here to scalameta/tutorial.